kubeyard/entrypoints/kubeyard.py

```diff
@@ -173,6 +177,10 @@ def deploy(**kwargs):
 
 @cli.command(help=DebugCommand.__doc__)
 @apply_common_options(initialized_repository_options)
+@click.argument(
+    'name',
+    required=False
```

Comment:
plz add help message

Reply:
I wanted to, but this `@click.argument` and there is no `help` arg. So I extended `DebugCommand.__doc__`. 

This line: https://github.com/socialwifi/kubeyard/pull/13/files#diff-961715217c8f29751b92791928bdca27R10